site/en/docs/extensions/mv3/architecture-overview/index.md

```diff
@@ -234,65 +234,33 @@ using [chrome.tabs.create()][api-create-tab] instead.
 
 For more information, explore the [Chrome API reference docs][api-reference].
 
-### Asynchronous vs. synchronous methods {: #async-sync }
+### Asynchronous methods {: #async-sync }
 
-#### Callbacks {: #callbacks }
-
-Most Chrome API methods are asynchronous: they return immediately without waiting for the operation
-to finish. If an extension needs to know the outcome of an asynchronous operation it can pass a
-callback function into the method. The callback is executed later, potentially much later, after the
-method returns.
+Most Chrome API methods are asynchronous; they return immediately without waiting for the operation to finish. If an extension needs to know the outcome of an asynchronous operation, there are two choices:
 
-A method is asynchronous when the callback parameter is available in its signature.
+* Use the returned promise.
+* Pass a callback function into the method.
 
-```js
-// Signature for an asynchronous method
-chrome.tabs.query(object queryInfo, function callback)
-```
+Note that these choiced are mutually exclusive. If you pass a callback to a method, no promise
```

Comment:
```suggestion
Note that these choices are mutually exclusive. If you pass a callback to a method, no promise
```

Reply:
Should this be an aside? 🤔 I think this is important not to miss, but you can make this call.

Reply:
I debated that myself. I think it's unlikely that anyone will actually try to use both methods, so I see no reason to draw attention to it.